platform/src/default.rs

```diff
@@ -197,4 +197,8 @@ impl Platform for DefaultPlatform {
             not_after: not_after_vec,
         })
     }
+
+    fn get_subject_alternative_name(&mut self) -> Result<SubjectAltName, PlatformError> {
```

Comment:
Might be good to populate this with something so we have test coverage/cert linting against the go code.

Reply:
Done except I just added it to tests in x509.rs so it is easier to test without using certifyKey.